src/data/sign.lean

```diff
@@ -332,6 +332,28 @@ end
 
 end add_group
 
+section linear_ordered_add_comm_group
+
+open_locale big_operators
+
+variables [linear_ordered_add_comm_group α]
+
+local attribute [instance] linear_ordered_add_comm_group.decidable_lt
```

Comment:
This feels a bit weird to me; do you know why `_root_.decidable_lt` doesn't fire here?

Reply:
No, but it's a known issue seen elsewhere in this file (and, thus, sometimes arising when using things from this file).

```lean
section linear_ordered_ring

variables [linear_ordered_ring α] {a b : α}

/- I'm not sure why this is necessary, see https://leanprover.zulipchat.com/#narrow/stream/
113488-general/topic/type.20class.20inference.20issues/near/276937942 -/
local attribute [instance] linear_ordered_ring.decidable_lt
```

Reply:
https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Decidable.20vs.20decidable_rel

Reply:
Let's just add a comment then and leave it as is for now then:
```suggestion
/- I'm not sure why this is necessary, see
https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Decidable.20vs.20decidable_rel -/
local attribute [instance] linear_ordered_add_comm_group.decidable_lt
```